eBay-brand-tracker/ebay_bt_venv/Lib/site-packages/fake_useragent/fake.py

```
@@ -0,0 +1,243 @@
+import random
+
+from fake_useragent import settings
+from fake_useragent.errors import FakeUserAgentError
+from fake_useragent.log import logger
+from fake_useragent.utils import load, str_types
+
+
+class FakeUserAgent:
+    def __init__(  # noqa: PLR0913
+        self,
+        browsers=["chrome", "edge", "firefox", "safari"],
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)